docker/core/Dockerfile

```diff
@@ -1,29 +1,35 @@
+# FIXME use alpine
```

Comment:
Why would alpine be better?

Reply:
not better but more "like a pro" (if you use lightweight containers is better, less footprint. Of course this is most important in systems we do not have, like kubernetes, but we want to be following good practices so it's good to mark this)

Reply:
Sounds good, then I would suggest making line 2 into `FROM node:14.15.1-alpine`

Reply:
yes I didn't do it myself because we don't need that level of lightweight-ness yet, and I therefore didn't want to risk breaking image building et al ('larger' image means that we have more/different tools, e.g. we do not use `apt-get` but `apk`, and i didn't want to spend time on that)